bash_completion

```diff
@@ -1618,9 +1618,10 @@ _comp_compgen_mac_addresses()
     # - ifconfig on Linux: HWaddr or ether
     # - ifconfig on FreeBSD: ether
     # - ip link: link/ether
+    # - long interface name will be truncated by ifconfig (#1089), prefer iproute2
```

Comment:
Usually, the long lines in the code that do not fit in 79 columns should be wrapped.

We haven't included references to the GitHub issues with just a number like \#1089. GitHub is not a unique and permanent platform for this project. It should better be a full link (`https://...`) or at least `GitHub\#1089` or `GitHub: #1089`. Existing cases are

https://github.com/scop/bash-completion/blob/b631277c23fb221fb2761f05ce2eb8457364bb96/bash_completion#L1063-L1065

https://github.com/scop/bash-completion/blob/b631277c23fb221fb2761f05ce2eb8457364bb96/bash_completion#L2562-L2563

https://github.com/scop/bash-completion/blob/b631277c23fb221fb2761f05ce2eb8457364bb96/bash_completion#L3050-L3051

Reply:
Also, this comment about interface name truncation is misplaced in the sense that the function is about generating mac addresses and I suppose it has nothing to do with interface names. So perhaps we should not do this change in `_comp_compgen_mac_addresses` at all -- or if we do, some commentary why we are doing it in the commit message would be in order.

Reply:
I've moved the comment inside `_comp_compgen_available_interfaces`.